crates/klave/src/https.rs

```
@@ -71,11 +71,19 @@ pub fn request(request: &Request<String>) -> Result<Response<String>, Box<dyn st
     };
 
     let mut parts = Response::new(String::new()).into_parts().0;
-    parts.headers = http_response.headers.iter().map(|header| (
-        header[0].parse::<http::header::HeaderName>().unwrap(),
-        header[1].parse::<http::header::HeaderValue>().unwrap()
-    )).collect();
-    parts.status = StatusCode::from_u16(http_response.status_code as u16).unwrap();
+    
+    parts.headers = http_response.headers.iter().filter_map(|header| {
```

This "swallows" any parsing error, which is *very* different from panicking as previously, and different from all other error treatment in the function, which is propagated.

Also, lines 58-68 could probably be simplified with the `?` operator (GitHub won't let me comment on something not in the changes).

---

Indeed, I was going toward the safe management of response handling, but here we want to avoid swallowing the error. For the moment I have rollback with what we had previously, this can be improved later.

I have updated lines 58-68 with the `?` operator.